### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by colinfrederickson**
Includes two items we discussed on 11/30 for inclusion into the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
 const results = await testSheet.testRecords(inputRows)
 expect(results).toMatchObject(expectedOutputRows)
 ```
+
+
+**CF Notes** - based on 11/30 convo do we want to include specifics on:
```
Comment (paddymul) [inline README.md:691]: I would add these to the FAQ section

File: README.md [hunk 2]
```diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
...
+
+-  Are there character limits for error messages?
```
Comment (paddymul) [inline README.md:693]: I don't know.  Want to try deploying a sheet with a reallllly long message and see where it fails.  test around 256 (smaller larger), 512, and 1024.  The backend might save the message and the frontend might do something wonky on display.

File: README.md [hunk 3]
```diff
@@ -686,3 +686,9 @@ const expectedOutputRows = [
...
+-  Addressing messaging around fields set via data hook (ex: we have a data hook that populates an empty field. Upon landing in review, that field is populated with a value and a message.  If another field on that record is populated, when data hooks are run again the field remains populated but without an error message letting the user know the field had been initially transformed.
```
Comment (paddymul) [inline README.md:694]: Add this to the FAQ for now.

This hints at the DataHook lifecycle, which is implied right now, but should be made more clear.

File: README.md (hunk 2 above)
Comment (colinfrederickson) [reply README.md]: this is more of a presentation issue on the front end, removing from FAQ.

**Pull request closed without merging**